livre/core/pipeline/Connection.h

```diff
@@ -1,4 +1,4 @@
-/* Copyright (c) 2011-2014, EPFL/Blue Brain Project
+/* Copyright (c) 2011-2016, EPFL/Blue Brain Project
```

Comment:
Is this file used? If you plan to move  the pipeline library out this isn't the right place for it.